xcode/Subconscious/Shared/AppTheme.swift

```diff
@@ -106,7 +106,7 @@ extension Color {
         .opacity(0.5)
     static let primaryButtonBackgroundDisabled = primaryButtonBackground
         .opacity(0.3)
-    static let primaryButtonText = SwiftUI.Color.accentColor
+    static var primaryButtonText: Color { SwiftUI.Color.accentColor }
```

Comment:
`accentColor` changes based on context, we want to preserve that context sensitivity.